### PR TITLE
CPLP-1103 add document content and status

### PR DESCRIPTION
**Pull request by Phil91**
Depending [PR](https://github.com/catenax-ng/product-portal-backend/pull/91#pullrequestreview-1012661324)

### CODE REVIEW

File: postgres/init/22-portal-content.sql [hunk 1]
```diff
@@ -557,7 +557,7 @@ ac1cf001-7fbc-1f2f-817f-bce058019993	607818be-4978-41f4-bf63-fa8d2de51154
 -- Data for Name: documents; Type: TABLE DATA; Schema: public; Owner: admin
 --
 
-COPY portal.documents (id, date_created, document, documenthash, documentname, document_type_id, company_user_id) FROM stdin;
+COPY portal.documents (id, date_created, document, documenthash, documentname, document_type_id, company_user_id, document_content, document_status_id) FROM stdin;
```
Comment (ntruchsess) [inline postgres/init/22-portal-content.sql:560]: documenthash -> document_hash
documentname -> document_name

**Review by ntruchsess — CHANGES_REQUESTED** [review]

**Review by ntruchsess — CHANGES_REQUESTED** [review]
21-portal-basedata.sql is missing the new docuement_status_id values. As this is an exported file we need to add the new table to backup_portal_basedata.sh (and the corresponding powershell-script as well) and do the export again.

File: postgres/init/20-portal.sql [hunk 2]
```diff
@@ -1,2066 +1,2100 @@
...
+--
+
+CREATE TABLE portal.documents (
+    id uuid NOT NULL,
+    date_created timestamp with time zone NOT NULL,
+    document_name character varying(255) NOT NULL,
+    document_type_id integer,
+    company_user_id uuid,
+    document_hash bytea,
+    document_content bytea DEFAULT '\x'::bytea NOT NULL,
```
Comment (ntruchsess) [inline postgres/init/20-portal.sql:539]: I notice we do not define default-values in any other table (besides company_assigned_apps), so here we should not do this here to be consistent.

**Review by ntruchsess — CHANGES_REQUESTED** [review]

File: postgres/init/20-portal.sql (hunk 2 above)
Comment (Phil91) [reply postgres/init/20-portal.sql]: i removed the default value.

File: postgres/init/22-portal-content.sql (hunk 1 above)
Comment (Phil91) [reply postgres/init/22-portal-content.sql]: done

**Review by ntruchsess — APPROVED** [review]

**Pull request merged**